### DISCUSSION TITLE
ARTIS v1.0.1 - Fishmeal

### DISCUSSION THREAD

**Discussion by theamarks** [Seafood-Globalization-Lab/artis-model#29] [Announcements]
## What's Changed
* Consumption update by @theamarks in https://github.com/Seafood-Globalization-Lab/artis-model/pull/28

ARTIS v1.0.1 now has `end_use` classifications based on specific HS6 product codes (`fishmeal`, `direct human consumption` and `other`) and better species resolution. Stay tuned for a full KNB data release of this data.

Change history - https://github.com/Seafood-Globalization-Lab/artis-model/compare/v1.0...v1.0.1

**Comment by mar-kk** [comment]
Is this version fully released as per now?

**Reply by theamarks** [reply]
Hi @mar-kk This version of the ARTIS model code is here on Github only. We have not run the full model with this fishmeal update yet. We hit a bug that has taken considerable time to sort out and we are on track to running the full model in April. We will need some time to run QAQC on the database and are aiming to have a new version of the ARTIS database available this summer with documentation on new features including fishmeal. We have been working on organizing our development protocols so moving forward we will only release model versions that are run fully. Happy to chat if you have any other questions.
